connector/connect/common/src/main/protobuf/spark/connect/expressions.proto

```diff
@@ -195,6 +197,17 @@ message Expression {
       DataType elementType = 1;
       repeated Literal element = 2;
     }
+
+    message Map {
+      DataType keyType = 1;
+      DataType valueType = 2;
+      map<string, Literal> map_data = 3;
```

Comment:
hmm... can the key of a map be ArrayMap/Struct?

My previous thought was to define keys and values as two arrays

Reply:
Yeah. So we can support the key as any literal.